src/index.js

```diff
@@ -2,13 +2,13 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 
 import App from './containers/App'
-import * as serviceWorker from './serviceWorker'
+import ErrorBoundary from './component/errorboundary/ErrorBoundary'
 
 import './scss/main.scss'
 
-ReactDOM.render(<App />, document.getElementById('root'))
-
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.unregister()
+ReactDOM.render(
+  <ErrorBoundary>
```

Comment:
I think the ErrorBoundary should live inside the App component and not on the index root file. This is because the index file should only now render a single component.  Could you please, move the `ErrorBoundary` to `App.js` component